force-app/main/userCommunity/lwc/hot_createRequestForm/hot_createRequestForm.js

```diff
@@ -524,6 +529,8 @@ export default class RecordFormCreateExample extends NavigationMixin(LightningEl
             if (this.currentRequestType === 'User') {
                 this.ordererForm = true;
                 this.userForm = true;
+            } else if (this.currentRequestType === 'Me') {
+                this.myRequest = true;
```

Comment:
Dette vil vel føre til at this.showInformationSharingText er true for Tyoe=='Me' (se else-setningen lenger ned)
Dette kan løses ved å flytte false-settingen til denne if-setningen.